docs/deployment_and_config.md

```diff
@@ -146,6 +146,14 @@ all:
     # Mordred Settings
     mordred_setups_repo_url: <repo_mordred_config.git>
 
+    # Ngninx Certbot
+    nginx_certbot_email: <nginx_certbot_email>
```

Comment:
This might be a little bit longer but users will understand what it is for.

```suggestion
    letsencrypt_register_email: <letsencrypt_register_email>
```